Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -481,7 +481,7 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
                 self.listView.scrollIndicatorInsets.top += self.typingIndicatorViewHeight
             }
 
-            if listView.isLastCellVisible {
+            if listView.isLastCellFullyVisible {
                 scrollToMostRecentMessage()
```

Comment:
one maybe unrelated idea how to easily improve the overall UX is to animate the typing indicator view in/out with slide animation which visually matches the inset update animation of the table view. Slack does it like this.

Reply:
Just tried it.

Slide in/out works nice if typing indicator is added to table view below the content so it becomes visible/hidden natively when inset is changed. 

But we can't keep it below the content because it's not visible if channel is scrolled up 🤔

We could have a bottom edge typing indicator constraint which we would animate together with insets but can't find correct animation options yet

Reply:
@VojtaStavik can we address this in a separate PR?

Reply:
For sure!! It was just a random idea :) I approved the PR